fix compatibilty with PyArrow 0.17.1, where FeatherReader was moved from pyarrow.feather to pyarrow.lib
author: Kenneth Hoste (HPC-UGent)
--- pyscenic-0.10.3/src/pyscenic/rnkdb.py.orig	2020-08-11 14:55:53.212511866 +0200
+++ pyscenic-0.10.3/src/pyscenic/rnkdb.py	2020-08-11 14:56:02.932604710 +0200
@@ -9,7 +9,11 @@
 from operator import itemgetter
 from .genesig import GeneSignature
 from cytoolz import memoize
-from pyarrow.feather import write_feather, FeatherReader
+from pyarrow.feather import write_feather
+try:
+    from pyarrow.feather import FeatherReader
+except ImportError:
+    from pyarrow.lib import FeatherReader
 from tqdm import tqdm
 
 
